modules/native/native-grammar.ym

```diff
@@ -36,13 +36,13 @@
 #include "syslog-names.h"
 #include "messages.h"
 #include "plugin.h"
-#include "cfg-grammar.h"
+// #include "cfg-grammar.h"
```

Comment:
I'm investigating this one as well because I can't build this module internally with Bison 3.5-3.6.